### ISSUE TITLE
Verbosity level is ignored for launchSettings.json

### ISSUE THREAD

**Issue by mtman** [dotnet/cli#9545]
## Steps to reproduce
```powershell
dotnet new console -n netcli-issue
cd .\netcli-issue\
mkdir Properties
'{"profiles": {"netcli-issue": {"commandName": "Project"}}}' > .\Properties\launchSettings.json
dotnet run --verbosity quiet
```

## Expected  output
Hello World!

## Actual output
Using launch settings from C:\doc\src\tmp\netcli-issue\netcli-issue\Properties\launchSettings.json...
Hello World!

## Environment data
`dotnet --info` output:

.NET Core SDK (reflecting any global.json):
 Version:   2.1.300
 Commit:    adab45bf0c

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17134
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.300\

Host (useful for support):
  Version: 2.1.0
  Commit:  caa7b7e2ba

.NET Core SDKs installed:
  2.1.201 [C:\Program Files\dotnet\sdk]
  2.1.300 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download

**Comment by peterhuene** [comment]
This is due to the `--verbosity` option only affecting MSBuild output, but the `run` command itself is outputting the `Using launch settings...` text.

Should be a simple fix to respect quiet verbosity level and have the command suppress this output.

**Comment by peterhuene** [comment]
I don't know how many times I'm going to put an issue/PR in 2.1.2xx when I mean 2.2.1xx :/

**Comment by peterhuene** [comment]
Hi @mtman.  Thanks for reporting this issue!  A fix has been made to the `release/2.2.1xx` branch and will ship with a future 2.2.100 version of the .NET Core SDK.  As such, I'm closing this issue as fixed.  Please re-open this issue if you have any questions or concerns.  Thanks!
